pkg/rehearse/jobs.go

```diff
@@ -282,26 +304,16 @@ func inlineCiOpConfig(container *v1.Container, ciopConfigs config.DataByFilename
 		}
 		filename := metadata.Basename()
 		loggers.Debug.WithField(logCiopConfigFile, filename).Debug("Rehearsal job uses ci-operator config ConfigMap, needed content will be inlined")
-		ciopConfig, ok := ciopConfigs[filename]
-		if !ok {
-			return fmt.Errorf("ci-operator config file %s was not found", filename)
-		}
-		ciopConfigResolved, err := registry.ResolveConfig(resolver, ciopConfig.Configuration)
-		if err != nil {
-			loggers.Job.WithError(err).Error("Failed resolve ReleaseBuildConfiguration")
-			return err
-		}
-
-		ciOpConfigContent, err := yaml.Marshal(ciopConfigResolved)
+		ciOpConfigContent, err := getResolvedConfigForTest(ciopConfigs, resolver, filename, testname)
 		if err != nil {
-			loggers.Job.WithError(err).Error("Failed to marshal ci-operator config file")
+			loggers.Job.WithError(err).Error("Failed to get resolved config for test")
 			return err
 		}
 
 		envs := container.Env
 		env := v1.EnvVar{
 			Name:  "CONFIG_SPEC",
-			Value: string(ciOpConfigContent),
+			Value: ciOpConfigContent,
```

Comment:
Could we add a sensible heuristic here to detect that we're likely going to overflow the allowed limit on `exec` and fail when we run the Pod?

Reply:
That would be quite difficult. The maximum varies based on the environment it's running in, and is the size of all of the command line arguments and all the environment variables combined.